sklearn/ensemble/forest.py

```diff
@@ -541,7 +571,7 @@ def predict(self, X):
 
             return predictions
 
-    def predict_proba(self, X):
+    def predict_proba(self, X, missing_mask):
```

Comment:
+1

and in general, I would even say, do we need to add this extra argument at all in forest? (I can understand for individual trees, not so much for forest where this could be hidden to the outside world)